css/style.css

```diff
@@ -0,0 +1,74 @@
+html
+{
+    text-align: center;
+    background-color: ghostwhite;
+ 
+}
+
+.log-in
+{
+    background: white;
+    border: 2px solid gray;
+    border-radius: 3px;
+    padding: 6px 24px;
+    font-size: 15px;
+}
+#sing-up:hover{
+    display: block;
+}
+.overlay {
+    top: 0;
+    right: 0;
+    bottom: 0;
```

Comment:
A lot of these could be done in bootsrap not to clog the css file.

Reply:
Thanks for the help